### DISCUSSION TITLE
Problem open Source-specific Multicast Stream (Windows 10)

### DISCUSSION THREAD

**Discussion by treffned** [smplayer-dev/smplayer#3] [Q&A]
Hello,

I have a Problem open a Source-specific Multicast Stream with SM-Player (Windows 10), if I try to open an Address like rtp://192.168.100.100@232.0.1.10:5004 the Stream does not open. In the IgmpV3 join Message, I can see, that for the Source “any” is requested. The Router is configured for SSM so he ignores the join to source any. I tried with VLC and in the IgmpV3 join Message the source is correct and it works. Do I miss here something or does the SM-Player not supports Source-Specific Multicast?

Best Regards

**Comment by smplayer-dev** [comment]
Send the MPlayer log after trying to play the video.

**Comment by treffned** [comment]
Hello, here is the Log
[smplayer_log.txt](https://github.com/smplayer-dev/smplayer/files/6152541/smplayer_log.txt)

**Reply by smplayer-dev** [reply]
Does it work if you open the URL directly in mpv?
`mpv.exe rtp://192.168.100.100@232.0.1.10:5004`

**Reply by treffned** [reply]
It doesn't work directly in mpv.

This is the IGMP Join send by mpv, as you can see there is "any" for the s´Source
![mpv](https://user-images.githubusercontent.com/80648977/111455406-09e58880-8716-11eb-8a79-ba94ca900459.png)

This is the IGMP Join send by VLC, there ist the Source filled and it works
![vlc](https://user-images.githubusercontent.com/80648977/111455405-09e58880-8716-11eb-88ab-d4efa74d934e.png)

**Reply by smplayer-dev** [reply]
Then I guess it might be a bug in mpv.

**Reply by treffned** [reply]
And this means for me I'm wrong here, because you "only" develop the Gui?

**Reply by smplayer-dev** [reply]
Yes, I only develop the gui. If this is a bug in mpv then maybe you could report the problem to its developers (https://github.com/mpv-player/mpv/issues). I'm afraid I can't fix problems in mpv or mplayer.
